tests/Components/Alerts/AlertTest.php

```diff
@@ -2,84 +2,65 @@
 
 declare(strict_types=1);
 
-namespace Tests\Components\Alerts;
+test('the component can be rendered', function () {
+    session()->flash('alert', 'Form was successfully submitted.');
 
-use PHPUnit\Framework\Attributes\Test;
-use Tests\Components\ComponentTestCase;
-
-class AlertTest extends ComponentTestCase
-{
-    #[Test]
-    public function the_component_can_be_rendered()
-    {
-        session()->flash('alert', 'Form was successfully submitted.');
-
-        $expected = <<<'HTML'
+    $expected = <<<'HTML'
             <div role="alert">
                 Form was successfully submitted.
             </div>
             HTML;
 
-        $this->assertComponentRenders($expected, '<x-alert/>');
-    }
-
-    #[Test]
-    public function we_can_specify_a_type()
-    {
-        session()->flash('error', 'Form contains some errors.');
+    assertComponentRenders($expected, '<x-alert/>');
+});
+test('we can specify a type', function () {
```

Comment:
Please leave one empty line between all tests.

Reply:
The Drift plugin makes the tests like that. I reviewed all the tests and added spaces.